airflow/jobs/scheduler_job.py

```diff
@@ -877,7 +877,7 @@ def process_file(self, file_path, zombies, pickle_dags=False, session=None):
         except Exception:
             self.log.exception("Error logging import errors!")
         try:
-            self.kill_zombies(zombies)
+            self.kill_zombies(dagbag, zombies)
```

Comment:
Surprised that our static checkers didn't catch this? Required arg was not passed !

Reply:
Yeah quite!

Reply:
Maybe pylint would have caught it but it is currently in **ToDo**:

https://github.com/apache/airflow/blob/6eaa7e3b1845644d5ec65a00a997f4029bec9628/scripts/ci/pylint_todo.txt#L11

Reply:
Indeed .

Reply:
Tomek has the PR #7484 in progress for that.